pkg/controllers/deprovisioning/validation.go

```diff
@@ -73,20 +77,20 @@ func (v *Validation) IsValid(ctx context.Context, cmd Command) (bool, error) {
 		case <-v.clock.After(waitDuration):
 		}
 	}
-
 	if len(v.validationCandidates) == 0 {
 		v.validationCandidates, err = GetCandidates(ctx, v.cluster, v.kubeClient, v.clock, v.cloudProvider, v.ShouldDeprovision)
 		if err != nil {
 			return false, fmt.Errorf("constructing validation candidates, %w", err)
 		}
 	}
+	// Compute PDBs and check if all the nodes are still deprovisionable.
+	pdbs, err := NewPDBLimits(ctx, v.kubeClient)
+	if err != nil {
+		return false, fmt.Errorf("tracking PodDisruptionBudgets, %w", err)
```

Comment:
```suggestion
		return false, fmt.Errorf("checking PodDisruptionBudgets, %w", err)
```

Reply:
I think tracking here makes more sense, as we're constructing the PDBLimits, rather than checking them against the set of pods on the node. WDYT?